reliability-v2/README.md

```diff
@@ -0,0 +1,212 @@
+# OpenShift V4 Reliability - V2
+## Introduction
+Software reliability testing is a field of software testing that relates to testing a software's ability to function, given environmental conditions, for a particular amount of time.
+Openshift Reliability testing is an operational testing scheme that uses a baseline work efficiency specification to evaluate the stability of openshift system in the given amount of time. The purpose is to discover problems in functionality. The baseline work efficiency specification was made up of daily tasks such as applications developing, hosting and scaling. 
+
+Reliability-v2 can simulate real world by concurrently running different tasks by multiple groups and users. 
+
+## Git
+```
+$ git clone git@github.com:openshift/svt.git
+cd svt/reliability-v2
+```
+
+## Install dependencies
+**NOTE**: Recommended to use a virtual environment(pyenv,venv) so as to prevent conflicts with already installed packages.
+```
+$ pip3 install -r requirements.txt
+```
+
+## Configuration
+Example [config](https://github.com/openshift/svt/blob/master/reliability-v2/config/example_reliability.yaml). 
+
+### Files needed
+
+If you installed your cluster with [Flexy-install](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/ocp-common/job/Flexy-install/), download kubeconfig, users.spec and kubeadmin-password files. If you installed cluster in other way, prepare the above files accordingly.
+
+Replace `kubeconfig` `kubeadmin_password` and `user_file` with above files in your config file.
+```yaml
+reliability:
+  kubeconfig: <absolute_path_to_kubeconfig>
+  users:
+    - kubeadmin_password: <path_to_kubeadmin-password>
+    - user_file: <path_to_users.spec>
+```
+
+For more detail explaination about the configuration, please go to [Configuration Detail](#Configuration-Detail).
+
+## Run
+
+### Run Reliability Test
+
+#### Integration
+[Slack Integration](#Slack-Integration) 
+
+If you want to receive notifications about the start stop of the test and errors happen during the Reliability test, configure Slack Integration before running the Reliability test.
+
+[Cerberus Integration](#Cerberus-Integration) 
+
+If you want to leverage Cerberus to check the healthy of the cluster and take action during the Reliability test, configure Cerberus Integration before running the Reliability test.
+
+[Kraken Integration](#Kraken-Integration)
+
+If you want to add [Kraken](https://github.com/cloud-bulldozer/kraken-hub) to inject errors during the Reliability test, configure Kraken Integration before running the Reliability test.
+
+Run the test
```

Comment:
I think we should move this sub-section above the integrations title

Reply:
Yeah, moving up makes it more clear.